### DISCUSSION TITLE
alist挂载pikpak，音频视频都无法用苹果手机自带浏览器进行预览

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [AlistGo/alist#5179] [General]
alist挂载pikpak，音频和视频都无法用苹果手机自带浏览器进行预览，播放音频就显示An audio error has occurred, player will skip forward in 2 seconds.有什么解决方案吗？或者能不能在下个一版本来解决这个问题

**Comment by Dashan-IZ** [comment]
期待大佬回复

**Comment by star2k** [comment]
说是safair浏览器的问题，不知道alist能否去兼容safari？
